Homework_7_Java_Arrays_Loops/JavaArraysAndLoops/src/StudentSearch.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+public class StudentSearch {
+    public static void main(String[] args) {
+        int[] studentIDs = {12345, 67890, 54321, 98765, 24680};
+
+        // Вводимо номер залікової книжки
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Введіть номер залікової книжки: ");
+        int gradebook = scanner.nextInt();
```

Comment:
also better variable name could be used -> for. ex. `gradebookId` to represent the purpose of the variable